Dockerfile

```diff
@@ -1,4 +1,14 @@
-FROM rust:bullseye AS builder
+FROM node:18 as ui-builder
```

Comment:
Nice. Can you do the same thing in the docker image?  
see https://github.com/quickwit-oss/quickwit/blob/main/Dockerfile#L25-L33